src/bl.cpp

```diff
@@ -156,6 +136,27 @@ void bl_init(void)
     Log_info("Non-GPIO wakeup (%d) -> didn't read buttons", wakeup_reason);
   }
 
+  Log_info("preferences start");
```

Comment:
I moved this down so that it's below `wait_for_serial()`. (otherwise I don't see it at all)